components/illustratedmessage/stories/illustratedmessage.stories.js

```diff
@@ -1,10 +1,8 @@
-import { Template as Link } from "@spectrum-css/link/stories/template.js";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { html } from "lit";
 import metadata from "../metadata/metadata.json";
 import packageJson from "../package.json";
 import { IllustratedMessageGroup } from "./illustratedmessage.test.js";
-import { Template } from "./template.js";
+import { SizingTemplate, Template } from "./template.js";
 
 /**
  * The Illustrated Message displays an illustration along with a heading and description. Optionally, part of the illustration can use an accent color. It can be used for status and errors, or as a call to action. For example, the Drop Zone component makes use of Illustrated Message as an area to drag and drop files.
```

Comment:
Could you update this description? For instance, accent color is no longer relevant, so we can remove that. I'm also noticing that we're not really talking about the button group either, it might be worth mentioning that it's part of the component.